### DISCUSSION TITLE
Access docs within a loaded model for topic merging?

### DISCUSSION THREAD

**Discussion by jburdo1** [MaartenGr/BERTopic#1225] [Q&A]
Is there a way to access the documents that went into training a model from within a freshly loaded model? I made a rookie mistake of not setting a seed when taking a random sample of documents before sentence embedding and model training. I saved the model before shutting down the kernel, but now that it's loaded back up and I went through and created a list of lists for manual topic reduction, I realized that I don't have the docs variable to call within 'merge_topics', and I can't retrain on the same docs because of my seed mistake. Not a fatal error, I can retrain after taking another (seeded) random sample and manually groups topics together again, but I thought I'd first ask if there's a better way. 

Thanks!

**Comment by MaartenGr** [comment]
Unfortunately, this is not possible. The documents are not saved within the model in order to keep the model small as saving input data is generally done outside of a ml model.

**Comment by jburdo1** [comment]
That was my assumption, thought I'd double check.

Thanks Maarten, keep up the great work!

Cheers,

Joe
---------------
Joseph Burdo, Ph.D.
Senior Scientist, RallyPoint
